lib/policies/retry.js

```diff
@@ -155,6 +155,7 @@ RetryPolicy.retryDecision = {
  * [RetryPolicy]{@link module:policies/retry~RetryPolicy} as child policy.
  * @extends module:policies/retry~RetryPolicy
  * @constructor
+ * @deprecated since 4.0 non-idempotent operations are never tried for write timeout or request error.
```

Comment:
Good call on deprecating and not removing it right away. Can we include a message to point to the correct policy, like "Use the default retry policy instead"?

Reply:
will do :+1: